src/h3lib/include/mathExtensions.h

```diff
@@ -20,12 +20,14 @@
 #ifndef MATHEXTENSIONS_H
 #define MATHEXTENSIONS_H
 
+#include <stdint.h>
+
 /**
  * MAX returns the maximum of two values.
  */
 #define MAX(a, b) (((a) > (b)) ? (a) : (b))
 
 // Internal functions
-int _ipow(int base, int exp);
+int64_t _ipow(uint64_t base, uint64_t exp);
```

Comment:
mismatch between uint64_t input and int64_t output? Why not both being signed?

Reply:
Updated!